src/openqdc/datasets/geom.py

```diff
@@ -59,6 +59,10 @@ class GEOM(BaseDataset):
     __name__ = "geom"
     __energy_methods__ = ["gfn2_xtb"]
```

Comment:
@FNTwin  Do still need a basis set for semi-empirical methods

Reply:
We don't to specify a basis set for semi empirical method as they are fully defined by their name. You don't get to choose the basis set as it is "builded inside" the technique itself.